weblate/trans/vcs.py

```diff
@@ -759,6 +759,30 @@ def get_remote_branch_name(self):
         else:
             return 'origin/{0}'.format(self.branch)
 
+    def configure_branch(self, branch):
+        """Configure repository branch."""
+        # Get List of current branches in local repository
+        # (we get additional * there indicating current branch)
+        branches = self.execute(['branch']).splitlines()
+        if '* {0}'.format(branch) in branches:
+            return
+
+        # Add branch
+        if branch not in branches:
+            self.execute(
+                ['checkout', '-b',  branch, 'origin/{0}'.format(branch)]
```

Comment:
The referenced SO question mentions refs/remotes/ here, are you sure origin is correct here?

Reply:
Yes, I am sure, on line 700, Subversion version just uses --prefix=origin/

Reply:
And this setting is particular for git svn, I don't think it can be propagated to more generic git version.